README.md

```diff
@@ -15,6 +15,10 @@ The OpenAPI Specification does not require rewriting existing APIs. It does not
 
 This GitHub project is the starting point for OpenAPI. Here you will find the information you need about the OpenAPI Specification, simple examples of what it looks like, and some general information regarding the project.
 
+Here is an example API description file following the OpenAPI Specification: https://petstore.swagger.io/v2/swagger.json
```

Comment:
As we have examples within the repo, I think it would be a mistake to link outside to one particular vendor's example.